dev/com.ibm.ws.jpa.tests.jpa_32_fat/test-applications/jakartadata/src/io/openliberty/jpa/data/tests/web/JakartaDataRecreateServlet.java

```diff
@@ -92,27 +93,25 @@ public void alwaysPasses() {
     }
 
     @Test
-    @Ignore("Reference issue: https://github.com/OpenLiberty/open-liberty/issues/28912")
     public void testOLGH28912() throws Exception {
```

Comment:
testOLGH28912 fails on Postgres; so the ignore needs to be replaced by a skip when Postgres

Reply:
okay